ubx/UBX/RXM.py

```diff
@@ -0,0 +1,133 @@
+"""Receiver Manager Messages: i.e. Satellite Status, RTC Status."""
+
+from ubx.UBXMessage import initMessageClass, addGet
+from ubx.Types import CH, U, U1, U4, I4, X4
+
+@initMessageClass
+class RXM:
+    """Message class RXM."""
+
+    _class = 0x02
+
+    class IMES:
+        """Indoor Messaging System information"""
+
+        _id = 0x61
+
+        class Fields:
+            pass
+
+    class IMES_GET:
+
+        _id = 0x61
+
+        class Fields:
+            numTx = U1(1)
+            version = U1(2)
+            reserved1 = U(3,2)
+            class Repeated:
+                reserved2 = U1(1)
+                txId = U1(2)
+                reserved3 = U(3,3)
+                cno = U1(4)
+                reserved4 = U(5,2)
+                doppler = I4(6)
+                position1_1 = X4(7, bitfield={
+                    "pos1Floor": {"s": 0, "e": 7},
+                    "pos1Lat": {"s": 8, "e": 30}})
+                position1_2 = X4(8, bitfield={
+                    "pos1Lon": {"s": 0, "e": 23},
+                    "pos1Valid": 24})
+                position2_1 = X4(9, bitfield={
+                    "pos2Floor": {"s": 0, "e": 8},
+                    "pos2Alt": {"s": 9, "e": 20},
+                    "pos2Acc": {"s": 21, "e": 22},
+                    "pos2Valid": 23})
+                lat = I4(10)
+                lon = I4(11)
+                shortIdFrame = X4(12,bitfield={
+                    "shortId": {"s": 0, "e": 11},
+                    "shortValid": 12,
+                    "shortBoundary": 13})
+                mediumIdLSB = U4(13)
+                mediumId2 = X4(14, bitfield={
+                    "mediumIdMSB": 0,
+                    "mediumValid": 1,
+                    "mediumboundary": 2})
+    @addGet
```

Comment:
Looks on wrong position. AddGet i think is used for classes which are used for sending and receiving with same fields.